cartography/intel/okta/oktaintel.py

```diff
@@ -0,0 +1,1193 @@
+# Okta intel module
```

Comment:
This module is far too large. Please decompose it into multiple modules.

Reply:
Also please stick to the organizational pattern used by the other intel modules (generally - API interaction at the top, transform below it, load below that, cleanup below that, and sync at the bottom) as it makes it much easier to navigate and understand the full module.

As is, this is incredibly difficult to review in full.